bin/commands/createFirstAdmin.js

```diff
@@ -74,21 +74,24 @@ function confirm (username, resetRoles) {
 
 function commandCreateFirstAdmin (options) {
   let
+    sdk,
     username,
     password,
     resetRoles;
 
   cout = new ColorOutput(options);
 
   process.stdin.setEncoding('utf8');
-  const action = {
-    controller: 'server',
-    action: 'adminExists'
-  };
 
-  return sendAction(action, options)
+  return getSdk(options)
+    .then(response => {
+      sdk = response;
+
+      return null;
```

Comment:
(nitpicking) can be simplified with `return sdk.server.adminExists();` (1 promise layer less)
Same goes for similar lines in other commands.

Reply:
I need to reuse the `sdk` few line below ([#L130](https://github.com/kuzzleio/kuzzle/pull/1319/files/aa23c1b1c4093be4f1f532ab645238fa7d3208b7#diff-30b2cf2a4cfe977c24b2f003ceb1c96cR130) to execute `security.createFirstAdmin`